sample-data/ParserTest.sol

```diff
@@ -109,6 +109,13 @@ contract ParserTest is IParserTest {
   /// @notice Separate line
   function _viewDuplicateTag() internal pure {
   }
+
+  /// fun fact: here are extra spaces after 1st return
```

Comment:
Could you move this function to `ParserTestFunny`? It's the contract that acts as a collection of weird natspec.

```suggestion
  /// fun fact: there are extra spaces after the 1st return
```